zeton/templates/caregiver_panel.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'index_child.html' %}
+{% block add_points %}
+<div class="text-center">
+    <h2>Dodawaj punkty</h2>
```

Comment:
Może zamiast "dodawaj" - "dodaj"?

Reply:
Nie wiem, jedyne zmiany jakie wprowadzam w tym branchu dotyczą usunięcia zbędnego kodu i tagów <center>. Nie zajmuję się treścią szablonów.

Reply:
poprawimy to w innym issue